src/prefect/deployments/runner.py

```diff
@@ -207,6 +209,10 @@ class RunnerDeployment(BaseModel):
             " a built runner."
         ),
     )
+    sla: Optional[Union[SlaTypes, list[SlaTypes]]] = Field(
```

Comment:
While this feature is experimental, I think it'd be worthwhile to prefix `sla` kwargs with an underscore to communicate that it's subject to change. @cicdw @zzstoatzz, do you think that's a reasonable approach for experimental kwargs, or is it enough to write in docstrings that kwargs are experimental and subject to change?
```suggestion
    _sla: Optional[Union[SlaTypes, list[SlaTypes]]] = Field(
```

Reply:
yep, that sounds good to me